ecc/enclave/enclave_stub.go

```diff
@@ -305,6 +309,10 @@ type StubImpl struct {
 
 // NewEnclave starts a new enclave
 func NewEnclave() Stub {
+	r := C.logging_set_callback(C.log_callback_f(C.golog_cgo_wrapper))
+	if r == false {
+		panic("error initializing logging for cgo")
+	}
```

Comment:
I have the feeling this would fit pretty well in a `init` method inside `logging.go`